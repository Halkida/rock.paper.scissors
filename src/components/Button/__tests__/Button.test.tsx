import { Button } from '../';
import * as renderer from 'react-test-renderer';

test('Button', () => {
  const component = renderer.create(
    <Button>Button</Button>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});