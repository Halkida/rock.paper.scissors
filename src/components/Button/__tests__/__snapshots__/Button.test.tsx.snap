// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button 1`] = `
<button
  className=" button button_default button_medium"
>
  Button
</button>
`;
